bindings/binding_typescript_wasm/__tests__/__snapshots__/transform.js.snap

```diff
@@ -118,75 +95,38 @@ exports[`transform in strip-only mode should strip type declarations 1`] = `
 `;
 
 exports[`transform in strip-only mode should throw an error when it encounters a module 1`] = `
-{
-  "code": "UnsupportedSyntax",
-  "message": "  x \`module\` keyword is not supported. Use \`namespace\` instead.
-   ,----
- 1 | module foo { }
-   : ^^^^^^
-   \`----
-",
-}
+"{"code":"UnsupportedSyntax","message":"\`module\` keyword is not supported. Use \`namespace\` instead.","snippet":"module","filename":"test.ts","line":1,"column":0}
```

Comment:
same here there is just module without ^ and foo

Reply:
Ah this is fixable from fast strip